### PR TITLE
2019 cu3 build information

### PR DESCRIPTION
**Pull request by alevyinroc**
## Type of Change

 - [ ] Bug fix (non-breaking change, fixes # )
 - [X] New feature (non-breaking change, adds functionality, fixes # )
 - [ ] Breaking change (effects multiple commands or functionality, fixes # )
 - [ ] Ran manual Pester test and has passed (`.\tests\manual.pester.ps1)
 - [ ] Adding code coverage to existing functionality
 - [ ] Pester test is included
 - [ ] If new file reference added for test, has is been added to github.com/sqlcollaborative/appveyor-lab ?
 - [ ] Nunit test is included
 - [ ] Documentation
 - [ ] Build system
 

### Purpose
 
Update build reference for latest SQL Server 2019 CU

### Approach

Updates internal buildref

### CODE REVIEW

File: bin/dbatools-buildref-index.json [hunk 1]
```diff
@@ -4196,6 +4196,11 @@
             "CU": "CU2",
             "Version": "15.0.4013",
             "KBList": "4536075"
+        },
+        {
+            "CU": "CU3",
+            "Version": "15.0.4023",
+            "KBList": "4538853"
         }
```
Comment (wsmelton) [inline bin/dbatools-buildref-index.json]: The date needs to be updated as well in the file.

Comment (wsmelton) [reply bin/dbatools-buildref-index.json]: Odd the test didn't fail, we validate the date in it as well.

**Comment by wsmelton** [comment]
@alevyinroc if you would not mind can you reset your fork and then the branch of this PR? The commit messages showing tell me it is out dated.

You can reset your branch to upstream using the commands below (this will keep your commit history cleaner):

```
git checkout development
git reset --hard upstream/development"
git push origin development --force
```

(Note: Don't do the reset --hard on your PR branch because you will loose changes you have made to the file.)

**Comment by jensenjk** [comment]
Not updated yet

**Comment by wsmelton** [comment]
...and that comment gets deleted.

File: bin/dbatools-buildref-index.json [hunk 2]
```diff
@@ -4181,7 +4181,7 @@
         {
             "Version": "15.0.2000",
             "SP": "RTM",
-            "SupportedUntil": "2025-07-01"
+            "SupportedUntil": "2030-01-08"
```
Comment (wsmelton) [inline bin/dbatools-buildref-index.json]: This should not have been touched, but wasn't paying attention so I'll fix it in another PR.

**Comment by alevyinroc** [comment]
@wsmelton sorry I didn't respond earlier. I've fixed my fork per your steps above so hopefully next time things will be cleaner. Looks like this one is all set, thanks.

**Comment by jensenjk** [comment]
Hi, something may have failed – the 2019 cu3 is still not on the list

[cid:image001.png@01D60347.80C1E380]

**Comment by wsmelton** [comment]
There is more to be done.

**Pull request merged**